### ISSUE TITLE
new static deployments are broken

### ISSUE THREAD

**Issue by ghost** [vercel/vercel#289]
Similar to #288

I was trying to deploy earlier but failed. Here is my environment. 
```
$ node --version
v6.3.0
$ now --version
𝚫 now 4.1.2
```
Using default deploy by `now`,  seems some issues were there cause the deployment took much longer with error showing `> FetchError: network timeout at: http://10.100.202.49:3000`. 🤔 Would be nice to have some investigation, if you need more information please let me know. 
```
$ now
> Deploying ~/try/xxx.xxx
> Using Node.js 7.0.0 (default)
> Ready! https://xxxxxxx-xwsmsifefl.now.sh (copied to clipboard) [2s]
> Initializing…
> Building
> ▲ npm install
> ⧗ Installing:
>  ‣ serve@^3.2.0
> FetchError: network timeout at: http://10.100.202.49:3000
> ▲ npm install
> Installing package serve@^3.2.0
> ▲ npm start
> > xxxx.xxx@ start /home/nowuser/src
> > serve ./content
> npm ERR! 
> npm ERR! Failed at the xxxx.xxx@ start script 'serve ./content'.
> npm ERR! Make sure you have the latest version of node.js and npm installed.
> npm ERR! If you do, this is most likely a problem with the xxxxx.xxx package,
> npm ERR! not with npm itself.
> npm ERR! Tell the author that this fails on your system:
> npm ERR!     serve ./content
> npm ERR! You can get information on how to open an issue for this project with:
> npm ERR!     npm bugs xxxx.xxx
> npm ERR! Or if that isn't available, you can get their info via:
> npm ERR!     npm owner ls xxxx.xxx
> npm ERR! There is likely additional logging output above.
> npm ERR! Linux 4.7.0-coreos
> npm ERR! argv "/usr/bin/node" "/usr/bin/npm" "run" "start"
> npm ERR! node v7.0.0
> npm ERR! code EACCES
> npm ERR! errno -13
> npm ERR! Error: EACCES: permission denied, open 'npm-debug.log.4282404183'
> npm ERR!  { Error: EACCES: permission denied, open 'npm-debug.log.4282404183'
> /home/nowuser/src/node_modules/877f43508f8b78f3e67c804b38705ae738d0118a/lib/listening.js:7
> const copyToClipboard = async text => {
>                               ^^^^
> SyntaxError: Unexpected identifier
>     at Object.exports.runInThisContext (vm.js:76:16)
>     at Module._compile (module.js:545:28)
>     at Module._extensions..js (module.js:582:10)
>     at Object.require.extensions.(anonymous function) [as .js] (/home/nowuser/src/node_modules/2a9909022d27ed3456706e4b2e3a923934cd710e/register.js:42:5)
>     at Module.load (module.js:490:32)
>     at tryModuleLoad (module.js:449:12)
>     at Function.Module._load (module.js:441:3)
>     at Module.require (module.js:500:17)
>     at require (internal/module.js:20:19)
>     at Object.<anonymous> (/home/nowuser/src/node_modules/877f43508f8b78f3e67c804b38705ae738d0118a/lib/index.js:11:19)
> npm ERR! Linux 4.7.0-coreos
> npm ERR! argv "/usr/bin/node" "/usr/bin/npm" "run" "start"
> npm ERR! node v7.0.0
> npm ERR! npm  v3.10.9
> npm ERR! code ELIFECYCLE
> npm ERR! xxxx.xxx@ start: `serve ./content`
> npm ERR! Exit status 1
> npm ERR! syscall open
> npm ERR!   syscall: 'open',
> npm ERR! 
> npm ERR! npm  v3.10.9
> npm ERR! path npm-debug.log.4282404183
> npm ERR!   errno: -13,
> npm ERR!   code: 'EACCES',
> npm ERR!   path: 'npm-debug.log.4282404183' }
> npm ERR! Please try running this command again as root/Administrator.
> npm ERR! Please include the following file with any support request:
> npm ERR!     /home/nowuser/src/npm-debug.log
```

**Comment by jamo** [comment]
Related: zeit/serve/issues/95

**Comment by jamo** [comment]
`npm i -g now@3.0.1` should sort it out before we can get `serve` fixed

**Comment by leo** [comment]
[Fixed!](https://github.com/zeit/now-cli/releases/tag/4.2.2)
